### DISCUSSION TITLE
The select box under the "Settings" title in site, something may be wrong with it

### DISCUSSION THREAD

**Discussion by babakfp** [fontsource/fontsource#970] [Q&A] [answered]
Hi. What is the deal with this select box under the settings title? I change between the options and nothing happens. Is it broken? Is the second option shouldn't be there? Something may be wrong with it.

<img width="935" alt="Screenshot 2024-05-15 221410" src="https://github.com/fontsource/fontsource/assets/44144724/a09c5424-2129-48dd-82e3-ac43820a21d3">

**Comment by ayuhito** [comment]
That is a bug, since Latin and Latin Extended are identical when it comes to previewing characters. Technically they are different underneath, but for the website we should only be displaying Latin.

**Reply by ayuhito** [reply, marked as answer]
Fixed in #971.
